### PR TITLE
Anova unequal sample

### PR DESCRIPTION
**Pull request by ahtavangar**
Hi David,

I have edited both anova1way functions to accept samples of unequal sizes. 

Thank you,

A

### CODE REVIEW

File: assessment/libs/stats.php [hunk 1]
```diff
@@ -2401,4 +2402,420 @@ function dotplot($a,$label,$dotspace=1,$labelspace=null,$width=300,$height=150)
...
+	//$args=func_get_args();
+	$mean=array();
+	$ss=array();
+	for($i=0;$i<$numargs;$i++){
+		$mean[$i]=mean($arr[$i]);
+		$ss[$i]=variance($arr[$i])*(count($arr[$i])-1);
+		//$n[$i]=count($args[$i]);
+	}
+	for ($i=0;$i<$numargs;$i++){
+		$gmean=array_sum($mean)/count($mean); //grand mean
```
Comment (drlippman) [inline assessment/libs/stats.php:2441]: This loop doesn't make sense to me.  It looks like you're calculating exactly the same thing multiple times.

File: assessment/libs/stats.php [hunk 2]
```diff
@@ -2401,4 +2402,420 @@ function dotplot($a,$label,$dotspace=1,$labelspace=null,$width=300,$height=150)
...
```
Comment (drlippman) [inline assessment/libs/stats.php:2511]: Ditto on above

Comment (ahtavangar) [reply assessment/libs/stats.php]: Oops. Yes, that doesn't do anything. Thanks for catching that.

**Comment by drlippman** [comment]
Thanks.  I'll get this merged manually.

**Pull request closed without merging**